### PR TITLE
More informative reset password messages. Fix #788

### LINKED ISSUE #788: Reset password should be more informative
* Add a note in the reset message and email that the link is only valid for 30 mins
* more informative error when can't reset than "COULD NOT RESET"

### PR DESCRIPTION
**Pull request by squirrelo**
This adds the 30 minute timeout information to the email sent and a more informative message when the user's password is not reset.

### CODE REVIEW

**Comment by wasade** [comment]
Maybe 24hr? I've never seen a site do 30 min for this before

**Comment by wasade** [comment]
Oh, I see, just expands the message

File: qiita_pet/handlers/user_handlers.py [hunk 1]
```diff
@@ -93,8 +93,11 @@ def post(self):
             info = user.info
             try:
                 send_email(user.id, "Qiita: Password Reset", "Please go to "
-                           "the following URL to reset your password: "
-                           "http://qiita.colorado.edu/auth/reset/%s" %
+                           "the following URL to reset your password: \n"
+                           "http://qiita.colorado.edu/auth/reset/%s  \nYou "
```
Comment (wasade) [inline qiita_pet/handlers/user_handlers.py]: The hostname here should be sourced from the configuration file as we cannot assume qiita is running from under colorado.edu

Comment (antgonza) [inline qiita_pet/handlers/user_handlers.py]: Agree, @squirrelo could you fix?

**Comment by squirrelo** [comment]
Good to go once tests pass

**Pull request merged**